pkg/graph/resolvers/apply_changes_test.go

```diff
@@ -0,0 +1,137 @@
+package resolvers
+
+import (
+	"testing"
+	"time"
+
+	"github.com/google/uuid"
+	_ "github.com/lib/pq" // required for postgres driver in sql
```

Comment:
Why is this import needed? I don't see any reference to anything dealing with SQL in this file.

Reply:
Good call out, it looks like it is leftover from old code. I removed it. Thanks!